test/views/helpers/index.js

```diff
@@ -0,0 +1,36 @@
+var assert = require('assert');
+var helpers = require('../../../views/helpers');
+
+
+describe('views.helpers', () => {
+  describe('#contrastColor', () => {
+    it('should return "white" when the value is not present', () => {
```

Comment:
In this case its not about duplication but about expressing the same shorter. My rule of thumb is: for a given `it "..."`, prepending all parent `describes` from the outer to the inner should result in a sentence. 

e.g. "views.helpers#contrastColor rgb color parsing supports '#' as prefix"